src/Bicep.Core.Samples/Files/Variables_LF/main.symbols.bicep

```diff
@@ -429,3 +429,55 @@ module.exports = function (context) {
+var loadedText2 = loadTextContent('Assets/TextFile.LF.txt')
+//@[4:15) Variable loadedText2. Type: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit.\n Donec laoreet sem tortor, ut dignissim ipsum ornare vel.\n  Duis ac ipsum turpis.\n\tMaecenas at condimentum dui.\n Suspendisse aliquet efficitur iaculis.\nIn hac habitasse platea dictumst.\nEtiam consectetur ut libero ac lobortis.\n\tNullam vitae auctor massa.\nFusce tincidunt urna purus, sit amet.\n'. Declaration start char: 0, length: 59
+
+var loadedTextInterpolation1 = 'Text: ${loadTextContent('Assets/TextFile.CRLF.txt')}'
+//@[4:28) Variable loadedTextInterpolation1. Type: string. Declaration start char: 0, length: 85
+var loadedTextInterpolation2 = 'Text: ${loadTextContent('Assets/TextFile.LF.txt')}'
+//@[4:28) Variable loadedTextInterpolation2. Type: string. Declaration start char: 0, length: 83
+
+var loadedTextObject1 = {
+//@[4:21) Variable loadedTextObject1. Type: object. Declaration start char: 0, length: 82
+  'text' : loadTextContent('Assets/TextFile.CRLF.txt')
+}
+var loadedTextObject2 = {
+//@[4:21) Variable loadedTextObject2. Type: object. Declaration start char: 0, length: 82
+  'text' : loadTextContent('Assets/TextFile.LF.txt')  
+}
+var loadedBinaryInObject = {
+//@[4:24) Variable loadedBinaryInObject. Type: object. Declaration start char: 0, length: 72
+  file: loadFileAsBase64('Assets/binary')
+}
+
+var loadedTextArray = [
+//@[4:19) Variable loadedTextArray. Type: array. Declaration start char: 0, length: 151
+  loadTextContent('Assets/TextFile.CRLF.txt')
+  loadTextContent('Assets/TextFile.LF.txt')
+  loadFileAsBase64('Assets/binary')
+]
+
+var loadedTextArrayInObject = {
+//@[4:27) Variable loadedTextArrayInObject. Type: object. Declaration start char: 0, length: 183
+  'files' : [
+    loadTextContent('Assets/TextFile.CRLF.txt')
+    loadTextContent('Assets/TextFile.LF.txt')
+    loadFileAsBase64('Assets/binary')
+  ]
+}
+
+var loadedTextArrayInObjectFunctions = {
+//@[4:36) Variable loadedTextArrayInObjectFunctions. Type: object. Declaration start char: 0, length: 270
+  'files' : [
+    length(loadTextContent('Assets/TextFile.CRLF.txt'))
+    sys.length(loadTextContent('Assets/TextFile.LF.txt'))
+    length(loadFileAsBase64('Assets/binary'))
+    sys.length(loadFileAsBase64('Assets/binary'))
+  ]
+}
+
+var loadedBinary = loadFileAsBase64('Assets/binary')
+//@[4:16) Variable loadedBinary. Type: Assets/binary. Declaration start char: 0, length: 52
```

Comment:
Type displayed for the binary file loaded is the relative path to the loaded file. It does not have apostrophes - so it's different from StringLiteral. I didn't include the Base64 content because 1) it might be very long and VSCode has limit for characters displayed, 2) base64 format wouldn't tell much user, which file he is referencing when hovered over variable. Therefore I decided to use file path, but the actual type is StringLiteral. 

Perhaps adding some indication that this is B64 format, eg. "Binary Base64: [FilePath]" might be better?